tikv-configuration-file.md

```diff
@@ -328,13 +328,6 @@ raftstore 相关的配置项。
 + 默认值：0
 + 最小值：0
 
-### `raft-max-size-per-message`
```

Comment:
Why is this removed?

Reply:
I'm not sure whether this config exists or not. 
I didn't find it in Tikv: etc/config-template.toml.
I alse search TiKV codebase for raft_max_size_per_message, still not found.

Reply:
Oh, I found it. This config is actually named as "raft_max_size_per_msg".